src/routes/Links.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+	import { page } from '$app/stores';
+	import { storeLightSwitch } from '@skeletonlabs/skeleton';
+	export let toggleNav;
+</script>
+
+<a
+	class:dark={$storeLightSwitch ? $page.url.pathname == '/' : ''}
+	class:active={$storeLightSwitch ? '' : $page.url.pathname == '/'}
+	data-sveltekit-preload-code
+	href="/"
+	on:click={toggleNav}>Home</a
+>
+<a
+	class:dark={$storeLightSwitch ? $page.url.pathname.startsWith('/about') : ''}
+	class:active={$storeLightSwitch ? '' : $page.url.pathname.startsWith('/about')}
+	data-sveltekit-preload-code
+	href="/about"
+	on:click={toggleNav}>About</a
+>
+<a
+	class:dark={$storeLightSwitch ? $page.url.pathname.startsWith('/events') : ''}
+	class:active={$storeLightSwitch ? '' : $page.url.pathname.startsWith('/events')}
+	data-sveltekit-preload-code
+	href="/events"
+	on:click={toggleNav}>Events</a
+>
+<a
+	class:dark={$storeLightSwitch ? $page.url.pathname.startsWith('/speakers') : ''}
+	class:active={$storeLightSwitch ? '' : $page.url.pathname.startsWith('/speakers')}
+	data-sveltekit-preload-code
+	href="/speakers"
+	on:click={toggleNav}
+>
+	Speakers
+</a>
+<a
+	class:dark={$storeLightSwitch ? $page.url.pathname.startsWith('/contact') : ''}
+	class:active={$storeLightSwitch ? '' : $page.url.pathname.startsWith('/contact')}
+	data-sveltekit-preload-code
+	href="/contact"
+	on:click={toggleNav}
+>
+	Contact
+</a>
+<a
+	class:dark={$storeLightSwitch ? $page.url.pathname.startsWith('/bubble') : ''}
+	class:active={$storeLightSwitch ? '' : $page.url.pathname.startsWith('/bubble')}
+	data-sveltekit-reload
+	href="/bubble-pop"
+	on:click={toggleNav}
+>
+	Game
+</a>
```

Comment:
Could these be put into a `{#each` block for each of the links passing in the various props here?

Something like:

```js
<script>
let links = [
	{href: `/`, name: `Home`},
	{href: `/about`, name: `About`},
	{href: `/events`, name: `Events`},
	{href: `/speakers`, name: `Speakers`},
	{href: `/contact`, name: `Contact`},
	{href: `/bubble-pop`, name: `Game`},
]
</script>

{#each links as {href, name}} 
	<a
		class:dark={$storeLightSwitch ? $page.url.pathname.startsWith({href}) : ''}
		class:active={$storeLightSwitch ? '' : $page.url.pathname.startsWith({href})}
		data-sveltekit-reload
		{href}
		on:click={toggleNav}
	>
		{name}
	</a>
{/each}
```

Reply:
Don't merge this just yet. I did some reworking with the light switch that will need to be reverted once they publish the next release of Skeleton. I'll do that next week.